libs/facilities/backend/insight-hub/src/lib/models/interfaces/time-series-request.interface.ts

```diff
@@ -0,0 +1,34 @@
+import { TimeSeriesOrdering } from '../enums';
+
+/**
+ * Interface for the parameters of requesting time series data
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export interface ITimeSeriesRequestParameter<SelectType> {
```

Comment:
Maybe add a comment on what the SelectType is (like an example) and add stricter type for selecttype 

```suggestion
export interface ITimeSeriesRequestParameter<SelectType extends Record<PropertyKey, any>> {
```